data/archiver/listener.py

```diff
@@ -34,28 +36,36 @@ def data_archiver_message_handler(self, body: str, msg: Message):
         return self.executor.submit(lambda: self._data_archiver_message_handler(body, msg))
 
     def _data_archiver_message_handler(self, body: str, msg: Message):
+        ingest_cli = Ingest()
         try:
             dict = json.loads(body)
             req = DataArchiverRequest.from_dict(dict)
             self.logger.info(f'Received data archiving request for submission uuid {req.sub_uuid}')
 
-            result = Archiver().start(req)
-            self.producer.publish(result.to_dict(),
-                exchange=self.pub_queue_config.exchange,
-                routing_key=self.pub_queue_config.routing_key,
-                retry=self.pub_queue_config.retry,
-                retry_policy=self.pub_queue_config.retry_policy)
-            
+            result = Archiver(ingest_cli, AwsS3()).start(req)            
             self.logger.info(f'Archived data for submission uuid {req.sub_uuid}')
             
 
         except ValueError as e:
-            self.logger.info(f'Invalid JSON request: {body}')
+            error_msg = f'Invalid data archiving request: {body}'
+            self.logger.info(error_msg)
```

Comment:
logger.exception would be better so that stack trace will be printed

Reply:
Indeed, `logger.exception` or `warning` or error`

Reply:
Ok, I think I know why I used logger.info instead of exception or error (not sure about warning). because this is a listener process, if I do logger.exception, it stops the message handler/process which we do not want if a message in the queue has an exception. we just want to display something in the log and carry on listening.